doc/tutorials/content/pcl_vcpkg_windows.rst

```diff
@@ -0,0 +1,145 @@
+* Google Benchmark - for benchmarks
+* OpenNI2
+* Realsense2
+* PNG - for a single openni app
+* Pcap - for Velodyne HDL driver
+
+PCL's optional dependencies not on VCPKG
+
+* CUDA - only a port that verify its installed (version 10.1).
+* GLUT
+* OpenNI
+* Ensenso
+* davidSDK
+* DSSDK
+* RSSDK
+
+
+Install PCL for usage
+=====================
+
+Running the following command in powershell in the VCPKG directory,
+will install PCL with default options as well as default triplet type (ie. x86).
+
+  ./vcpkg install pcl
+  
+
+.. note::
+
+  If there are new features or bugfixes that are not yet part of a release,
+  you can try to use --head, which downloads the master of PCL.
+  
+  - 2021-06-25 PCL 11(.1) doesn't support VTK 9, so if you need visualization module, use --head
+
+You can see the available PCL version and options in VCPKG `here <https://github.com/microsoft/vcpkg/blob/master/ports/pcl/vcpkg.json>`_.
+
+To enable specific features, you need to write:
+
+  ./vcpkg install pcl[qt,vtk]
+
+And all features:
+
+  ./vcpkg install pcl[*]
+
+If you want to install with a different triplet type, the easiest way is:
+
+  ./vcpkg install pcl --triplet triplet_type
+  
+ie.
+ 
+  ./vcpkg install pcl --triplet x64-windows
+
+This will acquire all the dependencies, build them and place the binaries
+in vcpkg/installed/triplet_type/bin for release and vcpkg/installed/triplet_type/debug/bin for debug.
+
+
+Using dependencies installed with VCPKG in CMake projects
+=========================================================
+
+Use `CMake <https://cmake.org/download>`_ to configure projects and remember to pass **vcpkg\\scripts\\buildsystems\\vcpkg.cmake** as toolchain file
+to enable CMake to find all the dependencies installed with VCPKG.
+
+See example below using the cmake window:
+
+.. list-table:: 
+
+    * - .. figure:: images/vcpkg/cmake_configure_1.png
+
+           Fig 1. Cmake configuration
+
+      - .. figure:: images/vcpkg/cmake_configure_2.png
+
+           Fig 2. Cmake configuration with vcpkg tool chain
+           
+
+Find PCL using CMake
+====================
+
+To use PCL in CMake project, take a look at Kunal Tyagi's minimal example `in this repository <https://github.com/kunaltyagi/pcl-cmake-minimum>`_
+
+
+Install PCL dependencies for contributions
+==========================================
+
+If you want to contribute to PCL, the easiest way to get dependencies
+using vcpkg is to run the install command from our `docker file <https://github.com/PointCloudLibrary/pcl/blob/master/.dev/docker/windows/Dockerfile>`_
+
+  ./vcpkg install dependencies_here --triplet triplet_type
+
+Remember to omit the *--clean-after-build*, as this removes the source code of the dependencies and limit debugging capabilities for those.
+
+To build PCL, you would have to get the `source <https://github.com/PointCloudLibrary/pcl>`_, preferably clone it using git.
+
+Use `CMake <https://cmake.org/download>`_ to configure what to build of PCL.
```

Comment:
to configure PCL as required

"what to build of PCL" shouldn't be needed and configuration also affects how to build, not just what to build.

Reply:
I left out 'as required' as it, at least for me, indicated that it should be done in a correct way and not necessarily as how the user wants/need it.